tests/models/test_semi_p.py

```diff
@@ -26,51 +26,37 @@
 from aepsych.likelihoods import BernoulliObjectiveLikelihood
 from aepsych.likelihoods.semi_p import LinearBernoulliLikelihood
 from aepsych.models import HadamardSemiPModel, SemiParametricGPModel
-from aepsych.models.inducing_point_allocators import AutoAllocator
+from aepsych.models.inducing_point_allocators import AutoAllocator, SobolAllocator
 from aepsych.models.semi_p import _hadamard_mvn_approx, semi_p_posterior_transform
 from aepsych.strategy import SequentialStrategy, Strategy
+from aepsych.utils import make_scaled_sobol
 from gpytorch.distributions import MultivariateNormal
 from parameterized import parameterized
 
 
 def _hadamard_model_constructor(
-    lb,
-    ub,
     stim_dim,
     floor,
     objective=FloorLogitObjective,
     inducing_point_method=AutoAllocator,
 ):
     return HadamardSemiPModel(
-        lb=lb,
-        ub=ub,
         stim_dim=stim_dim,
         likelihood=BernoulliObjectiveLikelihood(objective=objective(floor=floor)),
         inducing_size=10,
-        inducing_point_method=AutoAllocator(
-            bounds=torch.stack((torch.tensor(lb), torch.tensor(ub)))
-        ),
+        inducing_point_method=inducing_point_method,
```

Comment:
This isn't equivalent, the AutoAllocator in inducing_point_method isn't initialized so this violates the model init spec.